public/static/locales/eng/privacy-policy.md

```diff
@@ -2,108 +2,98 @@
 
 # Hvaða persónuupplýsingum safnar HR um þig og hver er tilgangurinn með söfnuninni?
 
-HR leggur áherslu á að vinna einungis þær persónuupplýsingar sem nauðsynlegar eru í samræmi við þann tilgang sem liggur að baki söfnun upplýsinganna. HR vinnur persónuupplýsingar ekki frekar í óskyldum tilgangi nema einstaklingi sé tilkynnt um slíkt og þá heimild sem vinnslan byggir á.
-HR safnar, eins og við á hverju sinni, einkum eftirfarandi persónuupplýsingum:
-
-• raddupptökum
-
-• lýðfræðilegum upplýsingum s.s. aldur, kyn og móðurmál.
-
-• samskiptaupplýsingum, s.s. tölvupósti
-
-• stafrænum fótsporum, s.s. nethegðun
-
-• upplýsingar tengdar samþykkis forsjáraðila fyrir barn s.s. kennitala barns og netfang forsjáraðila
+RU emphasizes the processing of only the personal information that is necessary in accordance with the purpose behind the collection of the information. RU does not further process personal information for unrelated purposes unless an individual is notified of this and the authorization on which the processing is based. RU collects, as applicable at any given time, the following personal information:
 
-• undir hvaða formerkjum þú, lest s.s. nafn skóla
+-   Voice recordings
+-   Demographic information e.g. age, gender and mother tongue.
+-   Contact information, if you create an account then we store your email.
+-   Digital cookies, e.g. online behaviour.
+-   Information related to the consent of custodians for the children participating, e.g. child's ID number and email address of the guardian.
+-   For which school you are reading for during the Samrómur reading competition.
 
-HR vinnur persónuupplýsingar einkum í þeim tilgangi að:
+RU processes personal information primarily for the purpose of:
 
-• búa til opið íslenskt raddgagnasafn til að styðja við og efla rannsóknir og þróun máltæknilausna
+-   creating an open Icelandic voice database to support and strengthen research and development of language technology solutions in Icelandic.
 
-Þegar þú notar vefinn [samrómur.is](https://samromur.is/) söfnum við upplýsingum um notkun þína, þ.e. IP tölu, tegund eða útgáfu vafra sem þú notar, tímasetningu og tímalengd heimsóknar og hvaða undirsíður þú heimsækir innan heimasíðu Samróms. Hér má finna nánari upplýsingar um notkun okkar á vafrakökum (e. cookies), [samrómur.is/vafrakökustefna](https://www.samromur.is/vafrakokustefna).
+When you use the website Samromur.is, we collect information about your use, i.e. IP address, type or version of browser you use, timing and duration of the visit and which subpages you visit within Samrómur website. Here you can find more information about our use of [cookies](https://www.samromur.is/vafrakokustefna).
```

Comment:
shouldn't the link to be the english document about the cookies, not the icelandic one? unless the english and icelandic ones are both at vafrakokustefna

Reply:
it is, the translation framework we are using does not add specific subpages for english/icelandic so the link is the same